x-pack/legacy/plugins/ml/public/access_denied/page.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiCallOut,
+  EuiPage,
+  EuiPageBody,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { NavigationMenu } from '../components/navigation_menu';
+
+export const AccessDeniedPage = () => (
+  <Fragment>
+    <NavigationMenu tabId="access-denied" />
+    <EuiPage>
+      <EuiPageBody>
+        <EuiPageContentHeader>
+          <EuiPageContentHeaderSection>
+            <EuiTitle>
+              <h1>
+                <FormattedMessage
+                  id="xpack.ml.management.jobsList.accessDeniedTitle"
+                  defaultMessage="Access denied"
+                />
+              </h1>
+            </EuiTitle>
+          </EuiPageContentHeaderSection>
+        </EuiPageContentHeader>
+        <EuiPageContentBody>
+          <EuiSpacer size="m" />
+          <EuiCallOut
+            title={i18n.translate('xpack.ml.accessDenied.label', {
+              defaultMessage: 'You need permission to access ML jobs',
```

Comment:
Compared to the previous angular version, the texts here don't mention the user roles anymore. However, the text is still specific about ML jobs, which could be irrelevant if a user doesn't have permissions to even access file visualizer. Maybe the messaging should be even more generic for this PR until we can come up with something more elaborate in combination with `react-router`?

Reply:
good point. i've updated the text.
It's worth noting that currently this page will not be redirected to automatically. So unless someone navigates to it directly in their browser, no one will see it.